RetailCoder.VBE/Settings/GeneralConfigProvider.cs

```diff
@@ -53,20 +53,21 @@ private void CheckForEventsToRaise(GeneralSettings other)
         protected virtual void OnLanguageChanged(EventArgs e)
         {
             var handler = LanguageChanged;
-            if (handler != null)
-            {
-                handler(this, e);
-            }
+            LanguageChanged?.Invoke(this, e);
```

Comment:
if this is indeed equivalent, then we can remove the local variable in all instances